### DISCUSSION TITLE
Access to iTunes Imported Playlists

### DISCUSSION THREAD

**Discussion by JeffKouba** [avantrec/soco-cli#21] [Q&A] [answered]
Is it possible to access iTunes imported playlists using Soco-cli?  Haven't figured how to do that.

**Comment by pwt** [comment]
Good question. I've never experimented with this, so I don't know the answer at the moment. I'll have a look in the next day or two.

**Comment by pwt** [comment, marked as answer]
I've now added this feature in **v0.3.43**. It was missing because I've never used imported playlists ... thanks for prompting me to look into it.

There are new actions to list library playlists, list the tracks in a library playlist, and to add a library playlist to the queue:
- `list_library_playlists` or `llp`
- `list_library_playlist_tracks <playlist_name>` or `llpt`
- `add_library_playlist_to_queue <playlist_name>` or `alpq`

These additions have only undergone light testing, so please do report any issues!

**Reply by JeffKouba** [reply]
Wow!  This works great!  Thanks!  I've tested on Windows 10 and on Unix on my Synology NAS and it works as documented.
I appreciate the time you spent on it!!

**Reply by pwt** [reply]
That's great -- thanks for testing.
